modules/Xsens_module/src/XsensRetargeting.cpp

```diff
@@ -179,8 +127,9 @@ bool XsensRetargeting::getJointValues()
 
     // in the msg thrift the first list is the joint names [get(0)], second the joint values
     // [get(1)]
-    yarp::os::Value humanjointsValueS = desiredHumanJoints->get(1);
-    yarp::os::Bottle* tmpHumanNewJointValues = humanjointsValueS.asList();
+
+    yarp::os::Value humanjointsValues = desiredHumanJoints->get(1);
+    yarp::os::Bottle* tmpHumanNewJointValues = humanjointsValues.asList();
```

Comment:
If the message is autogenerated from a thrift, you can import the library generated by Yarp and avoid to use the bottles.

Reply:
Thank you for the suggestion. If I add the dependency to the autogenerated library, it means `teleoperation` profile at build-time will have dependency to `human-dynamics` profile, which I would like to avoid it.

Reply:
ok!